File: Python/Product.py
class Product:
    def __init__(self, idProduct, name, brand, price):
        self.idProduct = idProduct
        self.name = name
        self.brand = brand
        self.price = price

    def get_idProduct(self):
        return self.idProduct

    def set_idProduct(self, idProduct):
        self.idProduct = idProduct

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_brand(self):
        return self.brand

    def set_brand(self, brand):
        self.brand = brand

    def get_price(self):
        return self.price

    def set_price(self, price):
        self.price = price

    def __str__(self):
        return "| {:<9d}| {:<15s}| {:<10s}| {:<6.2f} |".format(self.idProduct, self.name, self.brand, self.price)

# Contoh penggunaan kelas Product
# product = Product(1, "Product 1", "Brand 1", 10.50)
# print(product)

File: PHP/Product.php
<?php

class Product
{
    protected $idProduct;
    protected $name;
    protected $brand;
    protected $price;

    public function __construct($idProduct, $name, $brand, $price)
    {
        $this->idProduct = $idProduct;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    public function getIdProduct()
    {
        return $this->idProduct;
    }

    public function setIdProduct($idProduct)
    {
        $this->idProduct = $idProduct;
    }

    public function getName()
    {
        return $this->name;
    }

    public function setName($name)
    {
        $this->name = $name;
    }

    public function getBrand()
    {
        return $this->brand;
    }

    public function setBrand($brand)
    {
        $this->brand = $brand;
    }

    public function getPrice()
    {
        return $this->price;
    }

    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function __toString()
    {
        return sprintf("| %-9d| %-15s| %-10s| %-6.2f |", $this->idProduct, $this->name, $this->brand, $this->price);
    }
}
